### PR TITLE
feat: optional tags

### PR DESCRIPTION
**Pull request by arkbriar**
This is a breaking change that
- Turns off `tags.etcd` and `tags.minio` by default.
- Add a new tag `tags.bundle` to enable all the bundled sub-charts.

However it still maintains the semantics compatible. That means by explicitly providing the tags one can migrate from the old charts.

It also addressed an issue when `etcd` and `minio` are both disabled by explicitly include the `bitnami/common`.

### CODE REVIEW

**Comment by arkbriar** [comment]
Changes are a bit complicated so I'll add test cases in the following PRs before releasing a new release.

**Review by wjf3121 — APPROVED** [review]
LGTM!

**Comment by arkbriar** [comment]
> LGTM!

Thanks for the approval! 

Let me keep this open a bit longer cause I'd like to cover the cases with unit tests. Will merge it once that PR is approved as well.

**Comment by arkbriar** [comment]
Merge it now to unblock the CI for the following PRs...

**Pull request merged**
